resource-managers/kubernetes/integration-tests/src/test/scala/org/apache/spark/deploy/k8s/integrationtest/KubernetesSuite.scala

```diff
@@ -101,6 +101,32 @@ private[spark] class KubernetesSuite extends SparkFunSuite with BeforeAndAfter {
     runPySparkPiAndVerifyCompletion(PYSPARK_PI_CONTAINER_LOCAL_FILE_LOCATION, Seq.empty[String])
   }
 
+  test("Run SparkR Job on file locally") {
+    assume(testBackend.name == MINIKUBE_TEST_BACKEND)
+
+    launchStagingServer(SSLOptions(), None)
```

Comment:
Don't think we need a staging server if the file is in the image.

Reply:
Actually I just think the test is misnamed - looks like this test is shipping the file while the test below expects the file to be on the container.

Reply:
`src/test/R/dataframe.R` is what we are pushing up to the RSS, which is why it is being launched. Hmm, isn't that file locally stored? What would be the correct naming convention?